packages/lambda-tiler/src/routes/health.ts

```diff
@@ -0,0 +1,81 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import Sharp from 'sharp';
+import PixelMatch = require('pixelmatch');
+import { Epsg, Tile } from '@basemaps/geo';
+import { LambdaHttpResponse, LambdaContext, HttpHeader } from '@basemaps/lambda';
+import { ImageFormat } from '@basemaps/tiler';
+import { tile } from './tile';
+
+export function getExpectedTileName(projection: Epsg, tile: Tile, format: ImageFormat): string {
+    return path.join(
+        __dirname,
+        '..',
+        '..',
+        `static/expected_tile_${projection.code}_${tile.x}_${tile.y}_z${tile.z}.${format}`,
+    );
+}
+
+export const TestTiles = [
+    { projection: Epsg.Google, format: ImageFormat.PNG, testTile: { x: 252, y: 156, z: 8 }, buf: null as any },
+    { projection: Epsg.Google, format: ImageFormat.PNG, testTile: { x: 8073, y: 5130, z: 13 }, buf: null as any },
+    { projection: Epsg.Google, format: ImageFormat.PNG, testTile: { x: 32294, y: 20521, z: 15 }, buf: null as any },
```

Comment:
could we add a NZTM test

Reply:
Sure, I need to Config a test TileSets for health. Which one do you want to use for testing? still sentinel and gebco?

Reply:
I have setup NZTM gebco and sentinel in dev for testing.
https://dev.basemaps.linz.govt.nz/?i=health&p=EPSG:2193&#@-42.08509876,173.94222058,z3.5822